src/pages/Register.jsx

```diff
@@ -19,13 +22,29 @@ const Register = () => {
 
         try {
             await register(email, password);
+            // send welcome email
+            const templateParameters = { // these parameters are outlined in the template
+                recipient: email,
+            };
+            // send welcome email trigger
+            e.preventDefault();
+            emailjs.send('service_vcya2vm', 'welcome_1ywtamz', templateParameters, 'xZBCwlq2NxOnENizZ') // 'YOUR_SERVICE_ID', 'YOUR_TEMPLATE_ID', form.current, 'YOUR_PUBLIC_KEY'
+                .then(() => {
+                    alert("Your account has been created!")
+                }, () => {
+                    alert("Your account could not be created.")
+                });
+
```

Comment:
This code is repeating in Profile.jsx as well. Is there a reason we need it in both files?

Reply:
two different emails !